### ISSUE TITLE
reproduce the differnet

### ISSUE THREAD

**Issue by ZhangJiajun1995** [MediaBrain-SJTU/RegAD#16]
I use  the official source code to reproduce the Differnet in your papers. But my experimental result is lower than yours. I don't konw if you made changes to the details of the experimental settings. If it's convenient, can you share the code about this part? 

My email is 1434416800@qq.com

Thanks a lot.

**Comment by chaoqinhuang** [comment]
We use the official source code in [DifferNet](https://github.com/marco-rudolph/differnet) to reproduce the results. For fairness of comparison, we extend DifferNet to DifferNet+, where a pre-training procedure is added, i.e., data from multiple categories are used to pre-train the normalizing flow-based model for DifferNet. Then the model is finetuned using its original method with few-shot data from the target category. All you need to do is to modify the dataloader and add a pre-training procedure using its original training codes.
